tests/python/frontend/tflite/test_forward.py

```diff
@@ -161,6 +163,39 @@ def with_fused_activation_function(input_tensor, fn_name):
         return math_ops.tanh(input_tensor)
     raise AssertionError("Unknown fused_activation_function {}".format(fn_name))
 
+def _test_split(in_shape, axis, num_Splits, dtype):
+    '''internal split tester taking as parameters in_shape, number of tensors to split into
+       and dtype (data type)'''
+    np_data = np.random.uniform(-5, 5, size=in_shape).astype(dtype)
+    with tf.Graph().as_default():
+        in_data = array_ops.placeholder(shape=in_shape, dtype=dtype)
+        out = array_ops.split(in_data, num_Splits, axis=axis)
+        compare_tflite_with_tvm([np_data], ['Placeholder:0'],  [in_data], out,
+                                out_names=[f'out_{n}:0' for n in range(num_Splits)])
```

Comment:
Python 3.5 doesn't support this syntax (Currently we should make sure 3.5 could work). Please change it into:
```python
out_names = ['out_' + str(n) + ":0" for n in range(num_Splits)]
compare_tflite_with_tvm([np_data], ['Placeholder:0'],  [in_data], out,
                                          out_names=out_names)
```